howtos/tailscale_install.rst

```diff
@@ -0,0 +1,77 @@
+.. _install_tailscale:
+
+Tailscale install
+=================
+
+This How-to covers only the installation of the `Tailscale <https://tailscale.com/>`_ cloud service program.
+
+.. note::
+
+    Once installed, see our :ref:`Tailscale service <configure_tailscale>`
+    documentation for configuration and use.
+
+.. _install_tailscale_upstream:
+
+Upstream instructions
+---------------------
+
+This How-to is derived from, and secondary to,
+the canonical upstream instructions as provided by `Tailscale <https://tailscale.com/>`_ themselves.
+It is intended only to assist in providing relevant links and light guidance.
+
+.. note::
+
+    Rockstor is "Build on openSUSE", specifically **openSUSE Leap** or **Tumbleweed**: depending on the installer used.
```

Comment:
Minor typo:
```suggestion
    Rockstor is "Built on openSUSE", specifically **openSUSE Leap** or **Tumbleweed**: depending on the installer used.
```

Reply:
@FroggyFlox Thanks, and much appreciated. I'm always surprised at my missing these. I'd already corrected identical typos before submission also; and it's only a small How-to!